nixos/modules/services/accessibility/orca.nix

```
@@ -20,6 +20,7 @@ in
 
   config = mkIf cfg.enable {
     environment.systemPackages = [ cfg.package ];
+    systemd.services.display-manager.path = [ pkgs.orca ];
```

Shouldn't this be:

```suggestion
    systemd.services.display-manager.path = [ cfg.package ];
```

?